testsuite/forge-cli/src/main.rs

```diff
@@ -442,6 +437,25 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
                 10000,
                 Some(Duration::from_secs(240)),
             )),
+        "account_creation_state_sync" => config
+            .with_network_tests(&[&PerformanceBenchmarkWithFN])
+            .with_initial_validator_count(NonZeroUsize::new(5).unwrap())
+            .with_initial_fullnode_count(3)
+            .with_genesis_helm_config_fn(Arc::new(|helm_values| {
+                helm_values["chain"]["epoch_duration_secs"] = 1200.into();
+            }))
+            .with_emit_job(
+                EmitJobRequest::default()
+                    .mode(EmitJobMode::MaxLoad {
+                        mempool_backlog: 50000,
```

Comment:
50K is probably too much - I have seen the emitter destabilizing with so high load. Let's make it between 30 to 40k?